### PR TITLE
GCP upgrade: remove single quotes so that oreg_url would be substituted correctly

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

File: ci-operator/templates/cluster-launch-e2e-upgrade.yaml [hunk 1]
```diff
@@ -199,7 +199,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e 'oreg_url=${oreg_url}' \
+          -e oreg_url=${oreg_url} \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
```
Comment (bbguimaraes) [inline ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: Double-quote these two lines.

Also, the `upgrade` container below has similar code, let's change that too.

Comment (vrutkovs) [reply ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: okay, I really hope this won't expand '${component}' this time

Comment (vrutkovs) [reply ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: Done

File: ci-operator/templates/cluster-launch-e2e-upgrade.yaml [hunk 2]
```diff
@@ -199,7 +199,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e 'oreg_url=${oreg_url}' \
+          -e oreg_url="${oreg_url}" \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
```
Comment (bbguimaraes) [inline ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: Is the `TYPE` env variable still required?

Either use `${TYPE}` with double quotes or drop it from the container's `env` and use `${CLUSTER_TYPE}` with single quotes.

File: ci-operator/templates/cluster-launch-e2e-upgrade.yaml [hunk 3]
```diff
@@ -238,7 +238,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo="${RPM_REPO}" \
-          -e 'oreg_url=${IMAGE_FORMAT}' \
+          -e oreg_url="${IMAGE_FORMAT}" \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/upgrade.yml
```
Comment (bbguimaraes) [inline ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: Same here.

File: ci-operator/templates/cluster-launch-e2e-upgrade.yaml (hunk 2 above)
Comment (vrutkovs) [reply ci-operator/templates/cluster-launch-e2e-upgrade.yaml]: Its required for openshift-ansible's `entrypoint-provider` - https://github.com/openshift/openshift-ansible/blob/master/images/installer/root/usr/local/bin/entrypoint-provider#L28.

Replaced `${CLUSTER_TYPE}` with `"${TYPE}"` where possible

**Pull request merged**